web/core/components/issues/issue-layouts/list/block-root.tsx

```diff
@@ -122,7 +122,7 @@ export const IssueBlockRoot: FC<Props> = observer((props) => {
       <DropIndicator classNames={"absolute top-0 z-[2]"} isVisible={instruction === "DRAG_OVER"} />
       <RenderIfVisible
         key={`${issueId}`}
-        defaultHeight="3rem"
+        defaultHeight="inherit"
```

Comment:
Did you check how it affects virtualisation, scrolling and quick add?

Reply:
code should be fine but as Satish mentioned please check while virtualization while scrolling